src/pt/lermangaonline/src/eu/kanade/tachiyomi/extension/pt/lermangaonline/LerMangaOnline.kt

```diff
@@ -0,0 +1,130 @@
+package eu.kanade.tachiyomi.extension.pt.lermangaonline
+
+import eu.kanade.tachiyomi.extension.pt.lermangaonline.LerMangaOnlineFilters.GenresFilter
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class LerMangaOnline : ParsedHttpSource() {
+    override val name = "Ler Mangá Online"
+
+    override val baseUrl = "https://lermangaonline.com.br"
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 1, 1, TimeUnit.SECONDS)
+        .build()
+
+    private fun String.toChapterName() =
+        trim().replace("\\d{1,2}-\\d{1,2}-\\d{2,4}".toRegex(), "")
+
+    private fun String.toChapterNumber(): Float =
+        "\\d+".toRegex().find(trim())?.value?.toFloat() ?: 0F
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("div.capitulo")!!.text().toChapterName()
+        date_upload = (element.selectFirst("span")?.text() ?: "").toDate()
+        chapter_number = name.toChapterNumber()
+        setUrlWithoutDomain(element.absUrl("href"))
+    }
+
+    override fun chapterListSelector() = "div.capitulos a"
+
+    override fun imageUrlParse(document: Document) = ""
+
+    override fun getMangaUrl(manga: SManga): String = baseUrl + manga.url
```

Comment:
```suggestion
```

Not needed; this is mostly for sources that use a JSON API instead of scraping. The default implementation of this is already `baseUrl + manga.url`.